docs/sections/user_guide/yaml/components/make_hgrid.rst

```diff
@@ -0,0 +1,252 @@
+.. _make_hgrid_yaml:
+
+make_hgrid
+==========
+
+Structured YAML to run :ufs-utils:`make_hgrid<make-hgrid>` is validated by JSON Schema and requires the ``make_hgrid:`` block, described below. If ``make_hgrid`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Here is a prototype UW YAML ``make_hgrid:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/make_hgrid.yaml
+
+UW YAML for the ``make_hgrid:`` Block
+-------------------------------------
+
+execution:
+^^^^^^^^^^
+
+See :ref:`here <execution_yaml>` for details.
+
+run_dir:
+^^^^^^^^
+
+The path to the directory where ``make_hgrid`` will write its outputs.
+
+config:
+^^^^^^^
+
+Describes the required parameters to run a ``make_hgrid`` configuration. 
+
+
+grid_type:
+""""""""""
```

Comment:
It's not a problem specific to this PR, but I think the visual difference between
```
config:
^^^^^^
```
and
```
grid_type:
"""""""""
```
is negligible to non-existent. Looking at the rendered HTML, it's not at all clear to me that `grid_type` et al. are subordinate to `config`. Possibly we should reconsider our formatting to make this clearer. Maybe we could discuss this in an upcoming DSM or Group.